libs/nettrace/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/lf-edge/eve/libs/nettrace
+
+go 1.19
```

Comment:
Yes, changed to 1.18

Reply:
Actually, I noticed that pillar build may fail even with Go 1.18 features:
```
 => => # vendor/golang.org/x/sys/unix/syscall_unix.go:118:7: unsafe.Slice requires go1.17 or later (-lang was set to go1.16; check go.mod)                                            
 => => # vendor/golang.org/x/sys/unix/sysvshm_unix.go:33:7: unsafe.Slice requires go1.17 or later (-lang was set to go1.16; check go.mod)                                             
 => => # # github.com/onsi/gomega/internal                                                                                                                                            
 => => # vendor/github.com/onsi/gomega/internal/polling_signal_error.go:19:33: undeclared name: any (requires version go1.18 or later)                                                
 => => # vendor/github.com/onsi/gomega/internal/polling_signal_error.go:40:14: undeclared name: any (requires version go1.18 or later)                                                
 => => # vendor/github.com/onsi/gomega/internal/polling_signal_error.go:56:65: undeclared name: any (requires version go1.18 or later)    
```

I would upgrade Go version dependency from 1.16 to 1.18 [here](https://github.com/lf-edge/eve/blob/master/pkg/pillar/go.mod#L3) in my next PR so that we can start using new Golang features, like generics.